analytics/src/analytics/integrations/etldb/main.py

```diff
@@ -17,35 +19,38 @@
 VERBOSE = False
 
 
-def init_db() -> None:
+def initialize_database() -> None:
     """Initialize etl database."""
-    # define the path to the sql file
-    parent_path = Path(__file__).resolve().parent
-    sql_path = f"{parent_path}/create_etl_db.sql"
-
-    # read sql file
-    with open(sql_path) as f:
-        sql = f.read()
-
-    # execute sql
-    try:
-        db = EtlDb()
-        cursor = db.connection()
-        cursor.execute(
-            text(sql),
-        )
-        db.commit(cursor)
-    except (
-        RuntimeError,
-        ProgrammingError,
-        OperationalError,
-        InsufficientPrivilege,
-    ) as e:
-        message = f"FATAL: Failed to initialize db: {e}"
-        raise RuntimeError(message) from e
+    # get connection to database
+    etldb = EtlDb()
+    current_version = etldb.get_schema_version()
+    print(f"current schema version: {current_version}")
```

Comment:
Can you make it to print out each migration (file name, version, etc) one at a time as it's running. I just noticed that there's only one print statement being emitted in CI, which isn't enough to debug if migrations are broken.

Reply:
Yes, I had those in there and took them out after I saw your other comment about print statements, haha. I'll put them back in.

Reply:
Done

Reply:
Output looks like this now:
```
$ make init-db
=> Initializing the database schema
=====================================================
poetry run analytics etl initialize_database
initializing database
current schema version: 0
applying migration for schema version: 1
migration source file: /Users/david/Code/simpler-grants-gov/analytics/src/analytics/integrations/etldb/migrations/versions/0001_create_tables_etldb.sql
applying migration for schema version: 2
migration source file: /Users/david/Code/simpler-grants-gov/analytics/src/analytics/integrations/etldb/migrations/versions/0002_create_schema_versions.sql
applying migration for schema version: 3
migration source file: /Users/david/Code/simpler-grants-gov/analytics/src/analytics/integrations/etldb/migrations/versions/0003_alter_tables_set_default_timestamp.sql
total migrations applied: 3
new schema version: 3
done
```

Reply:
If there are no migrations to apply, it will look like this:
```
$ make init-db
=> Initializing the database schema
=====================================================
poetry run analytics etl initialize_database
initializing database
current schema version: 3
total migrations applied: 0
new schema version: 3
done
```